### ISSUE TITLE
Difference between yolov4.pth and yolov4.conv.137.pth

### ISSUE THREAD

**Issue by Wilson0925** [Tianxiaomo/pytorch-YOLOv4#469]
Hi, are they just two different pretrained models, or they relate to each other?

**Comment by beybars1** [comment]
Hi! 

Generally, we use **yolov4.pth**, that was trained on MSCOCO dataset, as the final weights file to detect objects. It is already trained/validated for optimality. It is a good way to test the YOLOv4 model in your specific task. 

The **yolov4.conv.137.pth** is similarly trained on MSCOCO dataset, but in this case the last layers were frozen and were not used. We just eliminate them since we want to use this pre-trained weights to be trained for our specific object detection task. It is a common task to use this file to train the model on your own dataset. By doing this, we get all the semantically rich information of low level patterns such as lines and circles as well as more complex patterns from the MSCOCO dataset, and use these knowledge to adapt the model for our task.

**Comment by hustCYQ** [comment]
> Hi!
> 
> Generally, we use **yolov4.pth**, that was trained on MSCOCO dataset, as the final weights file to detect objects. It is already trained/validated for optimality. It is a good way to test the YOLOv4 model in your specific task.
> 
> The **yolov4.conv.137.pth** is similarly trained on MSCOCO dataset, but in this case the last layers were frozen and were not used. We just eliminate them since we want to use this pre-trained weights to be trained for our specific object detection task. It is a common task to use this file to train the model on your own dataset. By doing this, we get all the semantically rich information of low level patterns such as lines and circles as well as more complex patterns from the MSCOCO dataset, and use these knowledge to adapt the model for our task.

Excuse me, I want to train and test the model on COCO. I use the pre-trained model yolov4.pth and set Cfg.use_darknet_cfg to False. Theoretically speaking, the loss function should be small and have a high AP at the beginning. In fact, the loss is very large. AP is close to 0. how should I adjust it?

**Comment by ingbeeedd** [comment]
There's a problem in code
